frontend/src/old-pages/Clusters/Actions.tsx

```diff
@@ -102,40 +105,44 @@ export default function Actions () {
     </DeleteDialog>
     <StopDialog clusterName={clusterName} />
     <SpaceBetween direction="horizontal" size="xs">
-      <Button className="action" disabled={clusterStatus === 'CREATING' || clusterStatus === 'DELETE_IN_PROGRESS' || clusterStatus === 'CREATE_FAILED' || !isAdmin()} variant="normal" onClick={editConfiguration} iconName={"edit"}> Edit</Button>
-      {fleetStatus === "STOPPED" && <Button className="action" variant="normal" onClick={startFleet} iconName={"caret-right-filled"}> Start</Button>}
+      <Button className="action" disabled={clusterStatus === 'CREATING' || clusterStatus === 'DELETE_IN_PROGRESS' || clusterStatus === 'CREATE_FAILED' || !isAdmin()} variant="normal" onClick={editConfiguration}>
+        <div className="container">
+          <Edit /> Edit
```

Comment:
chore: it would be good to extract the edited labels in the i18n file

Reply:
Agree, going to add them.